buildpack.md

```diff
@@ -972,11 +972,13 @@ If an `order` is specified, then `stacks` MUST NOT be specified.
 
 A buildpack descriptor that specifies `stacks` MUST describe a buildpack that implements the [Buildpack Interface](#buildpack-interface).
 
-Stack authors MUST choose a globally unique ID, for example: "io.buildpacks.mystack".
-
-The stack ID:
-- MUST only contain numbers, letters, and the characters `.`, `/`, and `-`.
-- MUST NOT be identical to any other stack ID when using a case-insensitive comparison.
+Each stack in `stacks` either:
+- MUST identify a compatible stack:
+   - `id` MUST be set to a valid stack ID.
```

Comment:
Should we just link to the platform spec where it specifies it?

Reply:
I think we have the following options:
1. We can update this with a deep link to https://github.com/buildpacks/spec/pull/154/files#diff-e603760990971da3f77be4bb8d77c3405098f006814fd8c054d2d15f395b8330R151 **if** we ship platform API 0.5 before buildpack API 0.5. There are two sub options here:
     1. ship platform API before merging and update _this_ PR with the link
     1. merge now and PR a small change adding the link later
1. We can update this with a shallow link that generically points at the platform spec now
1. Do nothing

I will happily proceed with any of the above.

Reply:
Let's go with a broken link.